bazel build //workbase:distribution failed with " Error: Cannot find module 'cfonts' "

`bazel build //workbase:distribution` failed with "Error: Cannot find module 'cfonts'":

```
$ bazel build //workbase:distribution

INFO: Analysed target //workbase:distribution (0 packages loaded, 0 targets configured).
INFO: Found 1 target...
ERROR: /Users/lolski/grakn.ai/grakn/workbase/BUILD.bazel:45:1: Executing genrule //workbase:distribution failed (Exit 1) bash failed: error executing command /bin/bash -c 'source external/bazel_tools/tools/genrule/genrule-setup.sh; workbase/builder/npm-run-build.sh bazel-out/darwin-fastbuild/genfiles/workbase/grakn-workbase.zip'

Use --sandbox_debug to see verbose messages from the sandbox
internal/modules/cjs/loader.js:582
    throw err;
    ^

Error: Cannot find module 'cfonts'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
    at Function.Module._load (internal/modules/cjs/loader.js:506:25)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/Users/lolski/grakn.ai/grakn/workbase/.electron-vue/build.js:7:17)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! grakn-workbase@1.4.0 build: `node .electron-vue/build.js && ./node_modules/electron-builder/out/cli/cli.js --publish never`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the grakn-workbase@1.4.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?
cp: workbase/build/*.zip: No such file or directory
/private/var/tmp/_bazel_lolski/84c573308050bc0236391991cc923148/sandbox/darwin-sandbox/1/execroot/grakn_core/workbase /private/var/tmp/_bazel_lolski/84c573308050bc0236391991cc923148/sandbox/darwin-sandbox/1/execroot/grakn_core

> grakn-workbase@1.4.0 build /private/var/tmp/_bazel_lolski/84c573308050bc0236391991cc923148/sandbox/darwin-sandbox/1/execroot/grakn_core/workbase
> node .electron-vue/build.js && ./node_modules/electron-builder/out/cli/cli.js --publish never

/private/var/tmp/_bazel_lolski/84c573308050bc0236391991cc923148/sandbox/darwin-sandbox/1/execroot/grakn_core
Target //workbase:distribution failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 1.302s, Critical Path: 0.62s, Remote (0.00% of the time): [queue: 0.00%, setup: 0.00%, process: 0.00%]
INFO: 0 processes.
FAILED: Build did NOT complete successfully

```

--- a/README.md
+++ b/README.md
@@ -54,7 +54,11 @@ You can visit the [Setup Guide](http://dev.grakn.ai/docs/get-started/setup-guide
 Grakn is built using [Bazel](https://bazel.build). To compile Grakn Core from source:
 1. Make sure you have Java 8 and Python 2 installed on your machine.
 2. [Install Bazel](https://docs.bazel.build/versions/master/install-os-x.html)
-3. Run:
+3. Prerequisite for building Workbase:
+```
+$ bazel run @nodejs//:npm install
+```
+4. Run:
 ```
 $ bazel build //...
 ```

